modules/installation-azure-confidential-vms.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-network-customizations
+
+:_content-type: PROCEDURE
+[id="installation-azure-confidential-vms_{context}"]
+= Enabling Confidential VMs
+
+:FeatureName: Confidential VM
+
+You can enable Confidential VMs for compute nodes, control plane nodes, or all nodes when installing your cluster. 
+
+include::snippets/technology-preview.adoc[]
+
+//commenting out the second encryption option until https://issues.redhat.com/browse/OCPBUGS-18379 is resolved
+////
+Confidential VMs can operate in two modes: 
+
+* Only encrypting the virtual machine guest state storage, which contains the security state of the virtual machine
+* Encrypting the virtual machine guest state storage and the operating system storage
+
+If you encrypt the operating system storage, you can use a platform-managed encryption key or a key you manage.
+////
+
+[IMPORTANT]
+====
+You can use confidential VMs with the following VM sizes:
+
+* DCasv5-series
+* DCadsv5-series
+* ECasv5-series
+* ECadsv5-series
+
+Confidential VMs are currently not supported on 64-bit ARM architectures.
+====
+
+.Prerequisites
+* You have created an `install-config.yaml` file.
+
+.Procedure
+
+* Use a text editor to edit the `install-config.yaml` file prior to deploying your cluster and add the following stanza:
++
+[source,yaml]
+----
+controlPlane: <1>
+  platform:
+    azure:
+      settings:
+        securityType: ConfidentialVM <2>
+        confidentialVM:
+          uefiSettings:
+            secureBoot: Enabled <3>
+            virtualizedTrustedPlatformModule: Enabled <4>
+      osDisk:
+        securityProfile:
+          securityEncryptionType: VMGuestStateOnly <5>
+----
+<1> Specify `controlPlane.platform.azure` or `compute.platform.azure` to deploy confidential VMs on only control plane or compute nodes respectively. Specify `platform.azure.defaultMachinePlatform` to deploy confidential VMs on all nodes.
+<2> Enable confidential VMs. 
+<3> Enable secure boot. For more information, see Azure's documentation about link:https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch#secure-boot[secure boot].
```

Comment:
Here and the next line, should rewrite to avoid possessive "Azure's". I usually write things like this by saying "in the Azure documentation"